x/compliance/keeper/msg_server_update_compliance_info.go

```diff
@@ -104,6 +104,10 @@ func (k msgServer) UpdateComplianceInfo(goCtx context.Context, msg *types.MsgUpd
 		complianceInfo.Transport = msg.Transport
 	}
 
+	if msg.Owner != "" {
```

Comment:
It seems it was an overlook to add Owner to Update Compliance .proto....
It should not be editable. 
I propose to leave the Owner field non-editable (do not include this code), and rather remove the Owner field from the documentation.